egs/fame/v2/sid

```diff
@@ -0,0 +1 @@
+../v1/sid
```

Comment:
I think this should point to ../../egs/sre08/v1/sid (the original sid directory)

Reply:
all symbolic links are consistent with sre08-sre10 examples.. v1/sid points to the original sid dir.

Reply:
Probably I made a mistake in egs/sre10/v2. I think it's more consistent with the ASR recipes to have all of the symbolic links pointing directly to where they come from, rather than a chain.

Reply:
Now they should be better..